vuepress/docs/next/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,278 +1,234 @@
 
-    -   TaskCommentService: defines service methods related to task
-        comment manipulation. It should be implemented if the task
-        comment is supported by the engine.
+`TaskDefinitionService`: defines service methods related to task definition. A task definition specifies which fields or columns are available for all task instances.
 
-    -   TaskAttachmentService: defines service methods to operate on
-        task attachments. It should be implemented if the engine
-        supports file attachment on the task.
+`TaskCommentService`: defines service methods related to task comment manipulation. It should be implemented if the task comment is supported by the engine.
 
-    -   TaskFormService: defines service methods for task form
-        operations, like retrieving the form definition and submitting a
-        form. The Form object can be used to render a form dynamically.
+`TaskAttachmentService`: defines service methods to operate on task attachments. It should be implemented if the engine supports file attachment on the task.
 
-    -   TaskLifecycleService: defines service methods related to the
-        task lifecycle. The lifecycle operations move the task from one
-        state to another.
+`TaskFormService`: defines service methods for task form operations, like retrieving the form definition and submitting a form. The Form object can be used to render a form dynamically.
 
-    -   TaskLifecycleBulkService: defines methods for bulk lifecycle
-        operations. Like the TaskLifecycleService, methods here move the
-        task from one state to another, but this interface works with
-        multiple tasks at a time.
+`TaskLifecycleService`: defines service methods related to the task lifecycle. The lifecycle operations move the task from one state to another.
 
-    -   ProcessService: defines service methods for process definitions
-        operations.
+`TaskLifecycleBulkService`: defines methods for bulk lifecycle operations. Like the TaskLifecycleService, methods here move the task from one state to another, but this interface works with multiple tasks at a time.
 
-    -   ProcessFormService: defines service methods for process form
-        operations, like retrieving the form definition and submitting a
-        form. The Form object can be used to render a form dynamically.
+`ProcessService`: defines service methods for process definitions operations.
 
-    -   GroupService: define service methods related to groups from the
-        BPM engine.
+`ProcessFormService`: defines service methods for process form operations, like retrieving the form definition and submitting a form. The Form object can be used to render a form dynamically.
 
-## Creating a new PDA MFE
+`GroupService`: defines service methods related to groups in the BPM engine.
 
-There are no limitations on the stack that could be used to create PDA
+## Creating a New PDA MFE
```

Comment:
I think we can remove both sections here - Creating a new PDA MFE and Communication between MFEs, We have tutorials for those with more information, http://localhost:8080/v6.3.2/tutorials/micro-frontends/react.html and http://localhost:8080/v6.3.2/tutorials/micro-frontends/communication.html